pageserver/src/config.rs

```diff
@@ -142,6 +148,10 @@ pub struct PageServerConf {
 
     /// Number of concurrent [`Tenant::gather_size_inputs`] allowed.
     pub concurrent_tenant_size_logical_size_queries: ConfigurableSemaphore,
+
+    // How often to collect metrics and send them to the metrics endpoint.
+    pub metric_collection_interval: Duration,
+    pub metric_collection_endpoint: String,
```

Comment:
`reqwest::Url` would work as well, then error would come sooner than the http send.

Reply:
Fixed. 
I also changed the logic here: disable metric collection, if no `metric_collection_endpoint` is explicitly provided.